extension/encoding/awslogsencodingextension/README.md

```diff
@@ -69,6 +69,21 @@ extensions:
     format: elb_access_log
 ```
 
+## Log Format Identification
+
+All logs processed by this extension are automatically tagged with an `awslogs_encoding.format` attribute at the scope level to identify the source format. The following format values are used:
```

Comment:
We already have format identifiers (the ones specified in config) - why are we introducing new ones? Can we use the same in both places? I prefer the names below, so maybe we update config to use them. Then we don't need a table here at all - we can just say that the value is the same as the configured format.

Reply:
I will use the same name as in the config then.  I was just aiming for something simpler in the names. I don't think I should change the format options as part of this PR. That would require updates to all the customer's using the encoding extension

Reply:
Agreed, let's do that separately - thanks

Reply:
@axw I defined a new constants package within the extension to extract all formats so I can use them both in internal and in awslogsencodingextension.